tensorboard/webapp/plugins/npmi/util/number_format.ts

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+export function formatLargeNumberComma(num: number | null): string {
```

Comment:
Hmm, two issues:

1. for utility like this, try to write a JSDOC and give example input/output. Especially with the regex, it is hard to understand.
2. `,` surely is not good for internationalization :) For Europeans, "." is the right delimiter, right? For that, try to use Intl. `Intl.NumberFormat().format(12312323) -> `"12,312,323"` for me.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat

Reply:
Cool, did not know about that.